lib/src/widgets/overlay.dart

```diff
@@ -763,10 +763,10 @@ class _TapTarget extends StatelessWidget {
 ///    inner area, so you do not have to worry about that.
 ///  * [extendBackground] will expand the background circle if necessary.
 ///    The radius will be increased until the content fits within the circle's area
-///    and a padding of [BackgroundContentLayoutDelegate.outerContentPadding] will be added.
+///    and a padding of 6 will be added.
```

Comment:
Of 4, then

Reply:
Well, I guess it was a seperate commit that fixed this 😄